tfjs-backend-webgpu/src/kernels/BatchNormal.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {KernelConfig, FusedBatchNorm, FusedBatchNormInputs,FusedBatchNormAttrs, Tensor} from '@tensorflow/tfjs-core';
```

Comment:
Missing FusedBatchNormInputs and FusedBatchNormAttrs? These shoube be defined in the tfjs-cores.

Reply:
It was defined in https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/kernel_names.ts#L36 and it has the same inputs as batchNorm so I reuse it

Reply:
We are trying to match TensorFlow for kernel names as much as possible - the batchNormalization kernel is called `FusedBatchNorm` in TensorFlow.